bin/fsl_imglob.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+#   imglob - expand list of image filenames
+#   Stephen Smith, Mark Jenkinson and Matthew Webster FMRIB Image Analysis Group
+#   Copyright (C) 2009 University of Oxford
```

Comment:
two things:

1. we should send the patches back to fsl. their new release will be based on py3, so hopefully they have patched this as well.

2. we don't bundle with nipype but download from an external location till it's fixed in fsl. or include in nipype.externals and put a check in the test to use this.

putting in `bin` makes this available to users on their path, and they could be using this version without knowing.